.circleci/config.yml

```diff
@@ -16,4 +16,4 @@ jobs:
       - run: aws s3 cp s3://launchdarkly-pastebin/ci/dotnet/LaunchDarkly.EventSource.snk LaunchDarkly.EventSource.snk
       - run: dotnet restore
       - run: dotnet build src/LaunchDarkly.EventSource -f netstandard1.4
-      - run: dotnet test test/LaunchDarkly.EventSource.Tests/LaunchDarkly.EventSource.Tests.csproj -f netcoreapp1.1
+      - run: dotnet test test/LaunchDarkly.EventSource.Tests/LaunchDarkly.EventSource.Tests.csproj -f netcoreapp2.0
```

Comment:
I had to change the _test_ project to .NET Core 2.0 because the mock server code uses some I/O classes that aren't available in 1.1. This doesn't affect how the library itself is built.